scripts/process-data/healthkitv2samples.R

```diff
@@ -125,6 +125,12 @@ rm(dataset,
    vars, 
    df, 
    participants_to_exclude,
+   criteria,
```

Comment:
I am removing the variables that are created by only this script/step of the pipeline, to avoid issues with other scripts that share variable names. There are some base variables from the `fetch-data` step that most of the downstream scripts use, so I can't remove all objects from the local environment.